troubleshoot/log.js

```diff
@@ -93,8 +78,23 @@ function formatRecordValue(value) {
             </details>`;
     }
 
+
+    // Handle object
+    if (typeof value === 'object') {
```

Comment:
This might lead to false positives since a lot of stuff is `object` but we can change that if it turns out to be a problem.

Reply:
I agree, but keep in mind that the data comes from deserialized JSON, which is quite limited with regards to the types that can come out of it. In the worst case, we get exceptions, that's fine.